src/components/EmailCollector.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+const EmailCollector: React.FC = () => {
```

Comment:
At some point move next to the search bar, perhaps? Not sure if it looks too crowded up there though.